src/main/java/stsjorbsmod/campfire/ThirstOption.java

```diff
@@ -0,0 +1,35 @@
+package stsjorbsmod.campfire;
+
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.ImageMaster;
+import com.megacrit.cardcrawl.ui.campfire.AbstractCampfireOption;
+import stsjorbsmod.effects.CampfireThirstEffect;
+
+import static stsjorbsmod.JorbsMod.makeID;
+
+public class ThirstOption extends AbstractCampfireOption {
+    public static final String[] TEXT;
+
+    public ThirstOption(boolean active) {
+        this.label = TEXT[0];
+        this.usable = active;
+        this.updateUsability(active);
+    }
+
+    public void updateUsability(boolean canUse) {
+        this.description = canUse ? TEXT[1] : TEXT[2];
+        // TODO: Replace with big boy art
+        this.img = ImageMaster.loadImage("images/ui/campfire/ritual.png");
```

Comment:
This is probably a little too often to be doing file IO; better would be to call `loadImage` once to initialize a static field with the Texture, then have this be `this.img = BUTTON_IMAGE;`.